easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1508,6 +1508,27 @@ def det_location_for(path, target_dir, soft_name, target_file):
     return target_path
 
 
+def cleanup_easyconfigs(paths):
+    """
+    Clean up easyconfigs (in place) by filtering out comments/buildstats included by EasyBuild in archived easyconfigs
+    (cfr. FileRepository.add_easyconfig in easybuild.tools.repository.filerepo)
+
+    :param paths: list of paths to easyconfigs to clean up
+    """
+    regexs = [
+        re.compile(r"^# Built with EasyBuild.*\n", re.M),
```

Comment:
On 2nd thought, the `re.M` is required, to ensure that also a comment like `# Built with EasyBuild...` is filtered out if it doesn't occur on the 1st line (maybe the easyconfig was manually edited afterwards...).